app/assets/scripts/views/emergency.js

```diff
@@ -217,6 +217,31 @@ class Emergency extends React.Component {
             <li>Expat delegates<span className='content-highlight'>{n(get(report, 'num_expats_delegates'))}</span></li>
           </ul>
         </div>
+
+        <h4 className='global-spacing-2-t clear'>Epidemic figures</h4>
```

Comment:
@GregoryHorvath don't we need a conditional of some sort here so this only shows if the emergency type is Epidemic? I don't think we want this showing for all emergencies.

Reply:
That is the correct logic @batpad